server/store/manifest.go

```diff
@@ -86,7 +78,7 @@ func (s *manifestStore) initGlobal(awscli aws.Client, bucket string, manifestsFi
 		case len(parts) == 2 && parts[0] == "file":
 			data, err = os.ReadFile(filepath.Join(assetPath, parts[1]))
 		case len(parts) == 2 && (parts[0] == "http" || parts[0] == "https"):
-			data, err = httputils.GetFromURL(loc)
+			data, err = httpOut.GetFromURL(loc, false)
```

Comment:
Is that the expected behaviour in a dev environment? Does it make sense to support URLs to localhost even for production, i.e. when a customer runs there app locally alongside the MM server?

Reply:
I fixed the dev mode support; as to production-local use case 2/5 that will be signaled with an appropriate flag, alongside devmode.